Sources/Networking/Operations/Handling/UserInfoAttributeParser.swift

```diff
@@ -17,11 +17,8 @@ enum UserInfoAttributeParser {
 
     static func attributesUserInfoFromResponse(response: [String: Any], statusCode: HTTPStatusCode) -> [String: Any] {
```

Comment:
This whole class will go away soon, will be superseded by the new `ErrorResponse`.